setup.py

```diff
@@ -255,9 +287,22 @@ def run(self):
     print(error)
     bdist_wheel = None
 
+
+class InstallCommand(InstallCommandBase):
+    def finalize_options(self):
+        ret = InstallCommandBase.finalize_options(self)
```

Comment:
Why this class is needed?
Our linter reports an error on this line. It says InstallCommandBase.finalize_options doesn't return any value.

Reply:
I guess we can try removing this and check if it works out...